python/cudf/cudf/core/groupby/groupby.py

```diff
@@ -1436,6 +1436,9 @@ def mult(df):
 
         if self._sort:
             result = result.sort_index()
+        if self._as_index is False:
+            result = result.reset_index()
+            result[None] = result.pop(0)
```

Comment:
```suggestion
            result = result.reset_index(drop=True)
```
Could we do this instead?

Reply:
When I try that, I get a `Series` still. We're trying to end up with a `DataFrame` with a column named `None`:

```
0    1.0
1    2.5
dtype: float64
```

vs
```
   y  None
0  0   1.0
1  1   2.5
```